assets/CommonAssemblyInfo.cs

```diff
@@ -1,5 +1,5 @@
 using System.Reflection;
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
This needs to stay fixed at `1.0.0.0` - we substitute it dynamically in the _AppVeyor_ build.